enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -186,6 +190,50 @@ async def start_kernel(self, *args, **kwargs):
         self.parent.kernel_session_manager.create_session(kernel_id, **kwargs)
         return kernel_id
 
+    async def restart_kernel(self, kernel_id):
+        kernel = self.get_kernel(kernel_id)
+        self.log.debug(f"Current value of the Kernel Restarting: {kernel.restarting}")
```

Comment:
Could we please lowercase `Kernel` and `Restarting` in its various places?  Might as well reference the variable names as well.
```suggestion
        self.log.debug(f"Current value of the 'kernel.restarting': {kernel.restarting}")
```